tooling-tests/src/test/resources/test-src/output/tool_test_generate_13/modules/generated_clients/multipleassociations_client.bal

```diff
@@ -0,0 +1,136 @@
+import ballerina/sql;
+import ballerinax/mysql;
+import ballerina/persist;
+import foo/tool_test_generate_13 as entities;
+
+public client class MultipleAssociationsClient {
```

Comment:
IMO didnt we decide to change the submodule name?

Reply:
updated in [2a21b9e](https://github.com/ballerina-platform/persist-tools/pull/23/commits/2a21b9e57ac70e4577606f3e3b9b3ae543c5af46)